js/bitso.js

```diff
@@ -813,6 +981,80 @@ module.exports = class bitso extends Exchange {
         return await this.privateDeleteOrdersOid (this.extend (request, params));
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitso#cancelOrders
+         * @description cancel multiple orders
+         * @param {[str]} ids order ids
+         * @param {str|undefined} symbol unified market symbol
+         * @param {dict} params extra parameters specific to the bitso api endpoint
+         * @returns {dict} an list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        if (!Array.isArray (ids)) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders() ids argument should be an array');
+        }
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        let oids = '';
+        for (let i = 0; i < ids.length; i++) {
+            oids += ids[i];
+            if (i < ids.length - 1) {
+                oids += ',';
+            }
+        }
```

Comment:
This loop is not necessary, we can transpile `.join(',)`:

```suggestion
        const oids = ids.join (',');
```